seqio/utils.py

```diff
@@ -49,6 +49,8 @@ def set_global_cache_dirs(global_cache_dirs):
 
 def add_global_cache_dirs(global_cache_dirs):
   global _GLOBAL_CACHE_DIRECTORIES
+  if isinstance(global_cache_dirs, str):
```

Comment:
Hi Colin! Could we instead add a pytype hint here to make it clear that it expects a list:
`def add_global_cache_dirs(global_cache_dirs:Sequence[str]):`
and pass a list from the caller?

Reply:
We could, but I don't think that would make it significantly less error-prone.

Reply:
Hopefully we can rely on pytype checkers at some point but I think for now, adding both a pytype annotation and handling the str case works.

Reply:
Done.